### PR TITLE
update smoke tests for October 2019 preview release

### PR DESCRIPTION
**Pull request by danieljurek**

### CODE REVIEW

**Comment by pakrym** [comment]
This PR is stale, can we close it and reopen later if required?

**Comment by danieljurek** [comment]
This is odd... I see no conflicts on my local fork w/r/t upstream master

**Review by heaths — CHANGES_REQUESTED** [review]
243 commits with over 3,700 changed files? Seems you have a bad merge or something. I'm seeing files added that have been there for a long time.

File: common/SmokeTests/SmokeTest/SmokeTest.csproj [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
 
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.0.0-preview.4" />
-    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.0.0-preview.3" />
-    <PackageReference Include="Azure.Security.Keyvault.Secrets" Version="4.0.0-preview.4" />
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.0.0-preview.3" />
-    <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.6.0" />
+    <PackageReference Include="Azure.Identity" Version="1.0.0-preview.5" />
+    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.0.0-preview.4" />
+    <PackageReference Include="Azure.Security.Keyvault.Secrets" Version="4.0.0-preview.5" />
```
Comment (heaths) [inline common/SmokeTests/SmokeTest/SmokeTest.csproj]: We GA'd. Why use the preview.5 version?

**Review by heaths — APPROVED** [review]

File: common/SmokeTests/SmokeTest/SmokeTest.csproj (hunk 1 above)
Comment (danieljurek) [reply common/SmokeTests/SmokeTest/SmokeTest.csproj]: This PR was left behind a while back. I wanted to make sure it was updated to show that the October preview also passed smoke testing. The GA smoke tests is coming in https://github.com/Azure/azure-sdk-for-net/issues/8630

**Pull request merged**